src/main/java/core/basesyntax/service/operation/ReturnOperationHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.service.FruitTransaction;
+import java.util.Map;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Map<String, Integer> storage) {
+        String fruit = transaction.getFruit();
+        int amount = transaction.getAmount();
+
+        storage.put(fruit, storage.getOrDefault(fruit, 0) + amount);
+    }
```

Comment:
This method modifies the storage by adding the amount of returned fruits. However, it does not validate the input data. What if the transaction amount is negative? What if the fruit is null or empty? Consider adding validation for these cases to prevent incorrect data from being added to the storage.